parser/parser.y

```diff
@@ -2448,6 +2451,7 @@ FunctionNameConflict:
 |	"UTC_DATE"
 |	"CURRENT_DATE"
 |	"VERSION"
+|	"INTERVAL" %prec lowerThanIntervalKeyword
```

Comment:
Is it ok to add "interval" to  https://github.com/pingcap/tidb/pull/2370/files#diff-27c45ca411f005e1b8796b12fb53e26cL60 ?